src/i18n/es.json

```diff
@@ -1,4 +1,8 @@
 {
-    "app.title": "Edita <code>src/App.js</code> y guarda para recargar la página",
-    "app.link": "Aprende React"
-}
\ No newline at end of file
+  "app.title": "Edita <code>src/App.js</code> y guarda para recargar la página",
+  "app.link": "Aprende React",
+  "footer.allRightReserved" : "Todos los derechos reservados",
+  "footer.iso" : "Certificación de Calidad ISO 9001:2008",
+  "footer.privacy": "Políticas de privacidad y legales.",
+  "footer.privacy_link": "https://fromdoppler.com/privacidad"
+}
```

Comment:
Should you not use a hierarchical representation of the JSON?

```
{
  "app": {
    "title": "Edita <code>src/App.js</code> y guarda para recargar la página",
    "link": "Aprende React",
  },
  "footer": {
    "allRightReserved" : "Todos los derechos reservados",
    "iso" : "Certificación de Calidad ISO 9001:2008",
    "privacy": "Políticas de privacidad y legales.",
    "privacy_link": "https://fromdoppler.com/privacidad"
  }
}
```

Reply:
Yes!

Reply:
i tried use with this format and doesn't work. I research more about that